src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/DatabaseCrudTests.ps1

```diff
@@ -102,10 +102,11 @@ function Test-CreateDatabaseInternal ($location = "westcentralus")
 function Test-CreateDatabaseWithSampleName
 {
 	# Setup
+	$location = "westcentralus"
```

Comment:
@emgu-ms please us the [`Get-Location`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Common.ps1#L549) function from our common code to get the location for the test. You can specify the location you want to use, and if it is available, it will be used. If not, one of the available locations will be used instead. This makes it easier for others to record tests in the future.